### DISCUSSION TITLE
Can GHAS webhooks link to actual failure in repo and not just the advisory page?

### DISCUSSION THREAD

**Discussion by ryberc** [community/community#60261] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

As it currently stands, the GHAS webhooks do not provide a link to the failure in the repo but to an advisory page. We use the webhooks for our ticketing system and it makes it tedious to patch the failures because the tickets don't link directly to the failure. Please include URLs to the actual failures in the webhook payloads so that we can include that in our tickets. Thanks.

**Comment by AbdrrahimDahmani** [comment]
the GHAS webhooks do not provide a link to the failure in the repo but to an advisory page

**Comment by ryberc** [comment]
Yes, I am aware of that. I stated that in my post. I WANT them to provide a link to the failure in the repo. That is what I am asking for as a product improvement.

-Ryan

[Text  Description automatically generated with medium confidence]
S. Ryan Berckmoes
Security Admin
***@***.******@***.***>
FranklinCovey<https://franklincovey.com/>  |  IT
801.817.4007
LINKEDIN<https://www.linkedin.com/company/franklincovey/mycompany/>  |  TWITTER<https://twitter.com/franklincovey>  |  FACEBOOK<https://www.facebook.com/FranklinCovey/>  |  INSTAGRAM<https://www.instagram.com/franklincovey/>

From: Abderrahim Dahmani ***@***.***>
Sent: Friday, July 7, 2023 5:59 AM
To: community/community ***@***.***>
Cc: Ryan Berckmoes ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [community/community] Can GHAS webhooks link to actual failure in repo and not just the advisory page? (Discussion #60261)

You don't often get email from ***@***.******@***.***>. Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

the GHAS webhooks do not provide a link to the failure in the repo but to an advisory page

-
